Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyEditorAPI_Internals.h

```diff
@@ -326,6 +327,12 @@ namespace AzToolsFramework
                 }
             }
 
+            if (m_widget)
+            {
+                // Reset existing widget attributes and properties before reuse
+                m_rpeHandler.ResetGUIToDefaults_Internal(m_widget);
+            }
```

Comment:
Does this mean the new reset logic only applicable to DPE workflow?

Reply:
Yes, it only applies to DPE.